packages/medusa-js/src/request.ts

```diff
@@ -2,9 +2,16 @@ import axios, { AxiosError, AxiosInstance } from "axios"
 import * as rax from "retry-axios"
 import { v4 as uuidv4 } from "uuid"
 
+const unAuthenticatedAdminEndpoints = {
+  "/admin/auth": true,
+  "/admin/users/password-token": true,
+  "/admin/users/reset-password": true,
+  "/admin/invites/accept": true,
+}
 export interface Config {
   baseUrl: string
   maxRetries: number
+  apiKey: string
```

Comment:
Make optional, so Typescript won't complain in the storefront when initting the client.